client/src/document/ingredients/examples.tsx

```diff
@@ -49,7 +49,6 @@ function RenderLiveSample({ example }) {
     <>
       <h4>Result</h4>
       <iframe
-        className="live-sample-frame"
```

Comment:
If you can put this back, https://github.com/mdn/yari/pull/2881 won't cause too much conflict.